pathway_indexer/parser.py

```diff
@@ -97,22 +97,53 @@ def has_changes(row):
     unchanged_html_files = html_df[~html_df["HasChanged"]]
 
     # Copy unchanged HTML files and remove them from input directory
-    for _, row in unchanged_html_files.iterrows():
-        print("Skipping unchanged file:", row["Filepath"])
-        pathname = os.path.basename(row["Filepath"]).replace(".html", ".md")
-        src_path = os.path.join(last_data_json["last_folder_crawl"], "out", "from_html", pathname)
-        dst_path = os.path.join(out_folder, "from_html", pathname)
-
-        os.makedirs(os.path.dirname(dst_path), exist_ok=True)
-        # Copy unchanged file as .md in out_folder
-        if os.path.exists(src_path):
-            shutil.copyfile(src_path, dst_path)
-            print(f"Copied {src_path} to {dst_path}")
-
-        # Remove unchanged file from input_directory
-        if os.path.exists(row["Filepath"]):
```

Comment:
if normalized src_path == normalized dest_path, do we want to still remove row["Filepath"]? If so, then instead of continuing above, make the condition if os.path.exists(src_path) and os.path.normpath(src_path) != os.path.normpath(dst_path)